app/views/sephora_style_guide/home/_checkbox_radio.slim

```diff
@@ -0,0 +1,46 @@
+article#checkbox-radio
+  h2.h2 Checkboxes & Radio
+
+  section
+    h3.h3 Checkbox
+    .row
+      .col-2
+        .option-link @click='toggleCheck'
+          span v-if='isCheckboxChecked'
+            = inline_svg 'sephora_style_guide/checkbox-checked.svg', class: 'checkbox'
+          span v-if='!isCheckboxChecked'
+            = inline_svg 'sephora_style_guide/checkbox.svg', class: 'checkbox'
+          label.option-label Default          
+      .col-2
+        .option-link.disabled
+          span
+            = inline_svg 'sephora_style_guide/checkbox-disabled.svg', class: 'checkbox'
+          label.option-label Disabled    
+      .col-2
+        .option-link.disabled
+          span
+            = inline_svg 'sephora_style_guide/checkbox-disabled-checked.svg', class: 'checkbox'
+          label.option-label Disabled Checked  
+    
+    br
+    section
+      h3.h3 Radio
+      .row
+        .col-2
+          .option-link @click='toggleSelect'
```

Comment:
When showing a radio, you need to show a group of them so you know that they actually work as radio.

Reply:
Also needs a proper container. Bootstrap uses `form-check` https://getbootstrap.com/docs/4.0/components/forms/